lib/hammer_cli_foreman/table_preferences.rb

```diff
@@ -0,0 +1,71 @@
+module HammerCLIForeman
+  class TablePreferences < HammerCLIForeman::Command
```

Comment:
Let's stick with our conventions:
```suggestion
  class TablePreference < HammerCLIForeman::Command
```